app/models/manageiq/providers/openstack/network_manager.rb

```diff
@@ -111,4 +112,25 @@ def create_network_router_queue(userid, options = {})
     }
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
+
+  def create_floating_ip(options)
+    FloatingIp.raw_create_floating_ip(self, options)
+  end
+
+  def create_floating_ip_queue(userid, options = {})
```

Comment:
this is basically copied create_network_router_queue, except for 2 strings. Should be extracted